charts/testnet-manager/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
I'd suggest to also include the option to reference the existing secret as it might be insecure to pass the values directly in the chart.

Reply:
The way this is is currently set up, it would not be easy to add support for an existing secret. As I would need to implement support for loading an external secret file inthe application code. So I think would be better to tackle this in a future chart upgrade.